lib/src/usage_impl_html.dart

```diff
@@ -25,6 +25,26 @@ Future<Analytics> createAnalytics(
   ));
 }
 
+class AnalyticsHtml extends AnalyticsImpl {
```

Comment:
I'm not sure this class needs to exist, we could follow the same pattern as usage_impl_flutter.dart and allocate an AnalyticsImpl in createAnalytics